packages/manager/apps/web/client/app/hosting/cdn/shared/add-edit-cache-rule/hosting-shared-add-edit-cache-rule.component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './hosting-shared-add-edit-cache-rule.controller';
+import template from './hosting-shared-add-edit-cache-rule.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    goBack: '<',
+    alertError: '<',
+  },
+  controllerAs: '$Ctrl'
```

Comment:
```suggestion
```

The default value is `$ctrl` and we prefer to use it.

Reply:
changed